tests/integration/helpers.py

```diff
@@ -60,12 +60,12 @@
         "config": {},
         "app_name": "mysql-router",
     },
-    # "postgresql-k8s": {
-    #     "charm": "pgbouncer-k8s",
-    #      "channel": "1/edge",
-    #      "config": {},
-    #      "app_name": "pgbouncer",
-    #  },
+    "postgresql-k8s": {
+        "charm": "pgbouncer-k8s",
+        "channel": "1/edge",
+        "config": {},
```

Comment:
should this also use the `testing` profile?

Reply:
@shayancanonical it does not have it: https://github.com/canonical/pgbouncer-k8s-operator/blob/main/config.yaml

Reply:
:+1: i saw `postgresql-k8s`, but didnt realize that it was a config for pgbouncer